packages/components/src/date-time/date/styles.ts

```diff
@@ -106,15 +125,14 @@ export const DayButton = styled( Button, {
 		props.hasEvents &&
 		`
 		::before {
-			background: ${ props.isSelected ? COLORS.white : COLORS.theme.accent };
+			border: 2px solid ${ props.isSelected ? COLORS.white : COLORS.theme.accent };
 			border-radius: ${ CONFIG.radiusRound };
+			box-sizing: border-box;
 			bottom: 2px;
```

Comment:
I noticed that since we added the focus border, it changes the metrics for the absolute `bottom: 2px` positioning and bumps the dot position every time the focus moves in and out. Maybe we should position it with `translateY`? `translateY(9px)` seems to work for me.

Reply:
I'll try that out and see how it goes